spin/02-json-api.md

```diff
@@ -144,12 +144,15 @@ function answer(): string {
 
 This code snippet defines a function that can be used to handle HTTP requests for the Magic 8 Ball. The function randomly selects one of the four possible answers and returns it as a JSON string. 
 
-Install any dependencies with `npm install` and then build and run the Spin app.
+Install the dependencies and then build and run the Spin app.
 
 ```bash
+npm install
 spin build --up
 ```
 
+> Note: `npm install` is only needed when adding new dependencies to the javascript function.
```

Comment:
It is needed even otherwise to get the `@fermyon/spin-sdk` package.

Reply:
Let me rephrase to "on additional builds".